addons/website/static/src/components/website_loader/website_loader.js

```diff
@@ -71,8 +73,12 @@ export class WebsiteLoader extends Component {
             (isVisible) => {
                 if (isVisible) {
                     window.addEventListener("beforeunload", this.showRefreshConfirmation);
+                    if (!this.state.selectedFeatures || this.state.selectedFeatures.length === 0) {
+                        this.initProgressBar();
+                    }
```

Comment:
This should be commented, I have no clue what this is: "if I have nothing to do, show a progress bar"? :thinking: